qiskit/pulse/commands/sample_pulse.py

```diff
@@ -15,6 +15,9 @@
 class SamplePulse(PulseCommand):
     """Container for functional pulse."""
 
+    # index of sample pulse
+    _pulseIndex = 0
```

Comment:
The naming of pulses is not just an issue for SamplePulses. We may wish to name any `PulseCommand`

Reply:
This does not have use right now but will as we extend the specification as well as adding parameters.

Reply:
Ok, now I understood your thought. Currently `name` of `PulseCommand` consistent with the command name of Qobj. I need to change the assembler as well to put names from the class info, and make `name` to be independent of command name.